### DISCUSSION TITLE
URL rewrites with vercel.json and SvelteKit

### DISCUSSION THREAD

**Discussion by tobiassern** [vercel/vercel#9486] [Help]
I am trying to use vercel.json to do url rewrites with SvelteKit, and can't get it to work in production.

Intended behavior; When visiting "/test" it should rewrite to "/_target".

```
{
    "rewrites": [
        {
            "source": "/test",
            "destination": "/_target"
        }
    ]
}
```

I use vercel CLI and when running "vercel dev" locally it works as intended. But when I push the code to production it is not working. When trying the exact same vercel.json file with Next it works both locally and in production. 

I have setup a very simple test example (basically just a starter SvelteKit with an added vercel.json). 

**Vercel domain:** https://svelte-rewrites.vercel.app/test
**GIthub repository:** https://github.com/tobiassern/svelte-rewrites

Any help is appreciated

**Comment by edgahan** [comment]
I seem to be hitting this same issue, I'll just add that if I replace `rewrites` with `redirects` it works which makes me believe the file itself is being used properly.
